mitmproxy/contentviews/image/view.py

```diff
@@ -37,19 +33,3 @@ def __call__(self, data, **metadata):
             parts = image_parser.parse_jpeg(data)
             fmt = base.format_dict(multidict.MultiDict(parts))
             return "%s image" % f, fmt
```

Comment:
We still need a fallback here. I propose an almost empty view that just states the `image_type`.